hudi-spark-datasource/hudi-spark-common/pom.xml

```diff
@@ -228,6 +228,19 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Hive -->
```

Comment:
If this is not added, the compilation will fail in the spark2 profile, because in `CreateHoodieTableCommand`'s `classOf[HoodieParquetInputFormat]` need to load `SelfDescribingInputFormatInterface`:

```scala
    val inputFormat = tableType match {
      case DataSourceWriteOptions.COW_TABLE_TYPE_OPT_VAL =>
        classOf[HoodieParquetInputFormat].getCanonicalName
      case DataSourceWriteOptions.MOR_TABLE_TYPE_OPT_VAL =>
        classOf[HoodieParquetRealtimeInputFormat].getCanonicalName
      case _=> throw new IllegalArgumentException(s"UnKnow table type:$tableType")
    }
```
Compile successfully with Spark3 profile, because in spark3, spark-hive_2.12 depends on hive-exec:

```text
[INFO] +- org.apache.spark:spark-hive_2.12:jar:3.4.1:provided
[INFO] |  +- org.apache.hive:hive-exec:jar:core:2.3.1:provided
```